blog/2024-01-24-embed-figma/index.mdx

```diff
@@ -0,0 +1,75 @@
+    sources={{
+      light: '/blog/2024-01-24-embed-figma/figma-light.png',
+      dark: '/blog/2024-01-24-embed-figma/figma-dark.png',
+    }}
+  />
+</p>
+
+## Improve your team workflows by combining Stately and Figma
+
+Embedding Figma into Stately can significantly streamline and enhance a team’s workflow.
+
+Being able to view mockups of the states you are discussing make it easy to _see_ what might be missing and identify potential improvements. It allows team members to view designs directly as part of the app logic itself.
+
+Two examples of how this can work are:
+
+- Engineers can import state machine code, then designers can fill out images for each state of the actual logic in the code.
+
+- Designers, PMs, and other stakeholders can create the expected behavior by embedding Figma frames into a Stately diagram, then engineers can continue adding the functionality from the automatically scaffolded code.
+
+This seamless integration can speed up the design process and ensure all team members are on the same page. Team members can easily share design details, run critiques, and iterate with people across different specializations.
+
+It also ensures that everyone is always looking at the most up-to-date design, as changes made in Figma are instantly reflected in the embedded view. And with [GitHub Sync](blog/2024-01-09-introducing-bidirectional-github-sync) enabled, changes to the code can even automatically update how the Figma frames are structured in Stately.
+
+## How to embed Figma frames
+
+Figma embedding is part of our [Pro plan](/docs/studio-pro-plan). We offer a trial period if you would like to try it out.
+
+To embed Figma frames in Stately, first you need to authenticate using a personal access token.
+
+1. [Get a personal access token](https://help.figma.com/hc/en-us/articles/8085703771159-Manage-personal-access-tokens) from Figma.
+2. Enter this token in Stately by opening a project, finding the **Stately menu** in the top left, selecting **Account**, opening **Settings**, and clicking the **Figma** tab.
+
+Then you can link frames to states.
+
+3. In Figma, right click on the frame you want to embed, select **Copy/Paste as**, then **Copy link”.
+4. In Stately, select a state, open the **+** menu, choose **Embed Figma**, and paste your link.
+
+[Watch a video of how embedding Figma frames works](https://www.youtube.com/live/c05bOPeinSA?si=qyUlnx7f7_StLDZ_&t=1006) from our recent Office Hours.
+
+![Frame inside Figma showing the menu for copy/paste as, and copy link. Alongside a frame in Stately showing the plus menu and the Embed Figma option.](figmatostately.png)
```

Comment:
We could add a light mode/dark mode version of this image.

Reply:
I wish we had a way to click to view a larger, clearer version of this image since the text in the menus is so small and unclear. For now, the closest thing is right click and choose Open image in a new tab.